src/script/common/c_content.cpp

```diff
@@ -141,6 +142,10 @@ void push_item_definition_full(lua_State *L, const ItemDefinition &i)
 	lua_setfield(L, -2, "name");
 	lua_pushstring(L, i.description.c_str());
 	lua_setfield(L, -2, "description");
+	if (!i.detailed_description.empty()) {
+		lua_pushstring(L, i.detailed_description.c_str());
+		lua_setfield(L, -2, "detailed_description");
+	}
```

Comment:
Should do this even if the `detailed_description` is empty?

Reply:
This field should only be set when it isn't empty (like all other text fields, AFAIK). If this field isn't explicitly set, this would be `nil`, which is the correct behaviour.